Userland/Libraries/LibGfx/ImageFormats/JPEG2000Loader.h

```diff
@@ -10,6 +10,30 @@
 
 namespace Gfx {
 
+namespace JPEG2000 {
+
+struct TagTreeNode;
+class TagTree {
+public:
+    TagTree(TagTree&&);
+    ~TagTree();
+
+    TagTree(TagTree const&) = delete;
+    TagTree& operator=(TagTree const&) = delete;
```

Comment:
Any reason why we shouldn't be able to copy a `TagTree`? I assume removing these would also allow us to use the implicitly defined constructors and destructors.

Reply:
The default copy ctor won't work, because that'll copy the vector but not the `children` pointers of the `TagTreeNode`s. Since I don't need copying, I just deleted the copy ctor.